notifications/notifications.go

```diff
@@ -0,0 +1,30 @@
+package notifications
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+)
+
+var fcmUrl = flag.String("fcm", "http://fcmUrl", "Google Firbase Cloud Messaging URL")
+
+func SendNotification(title string, message string) error {
```

Comment:
warning: exported function SendNotification should have comment or be unexported (golint)